README.md

```diff
@@ -193,7 +193,7 @@ function | purpose
 `get_model_name(app_label, model)` | Gets the name of the model the swappable model has been swapped for (or the name of the original model if not swapped.)
 `get_model_names(app_label, models)` | Match a list of model names to their swapped versions.  All of the models should be from the same app (though their swapped versions need not be).
 `load_model(app_label, model, required=True)` | Load the swapped model class for a swappable model (or the original model if it hasn't been swapped).  If your code can function without the specified model, set `required = False`.
-`dependency(app_label, model)` | Generate a dependency tuple for use in Django 1.7+ migrations.
+`dependency(app_label, model, latest=False)` | Generate a dependency tuple for use in Django 1.7+ migrations. Use `latest=True` only when the default dependency doesn't work.
```

Comment:
only when depending on the first migration of the target dependency doesn't work (eg: when all migrations of the target module should be run), please keep in mind that using `latestTrue` can have [drawbacks](LINK-HERE).

Please link the django ticket indicated in the issue.